apps/widget-e2e/src/describer/modelAssessment/dataExplorer/describeAxisConfigDialog.ts

```diff
@@ -0,0 +1,135 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+export function describeAxisConfigDialog(
+  defaultXAxis: string,
+  defaultYAxis: string,
+  noY: boolean,
+  hasColorAxis: boolean
+): void {
+  describe("Axis settings dialog", () => {
+    describe("Y Axis settings dialog", () => {
+      it("should display settings dialog", () => {
+        cy.get(
+          '#DatasetExplorerChart div[class*="rotatedVerticalBox"] button'
+        ).click();
+        cy.get("#AxisConfigPanel div.ms-Panel-main").should("exist");
+        cy.get('button:contains("Cancel")').click();
+      });
+      it("should be able to hide settings", () => {
+        cy.get(
+          '#DatasetExplorerChart div[class*="rotatedVerticalBox"] button'
+        ).click();
+        cy.get("#AxisConfigPanel button.ms-Panel-closeButton").click();
+        cy.get("#AxisConfigPanel div.ms-Panel-main").should("not.exist");
+      });
+      it("should display right y-axis title", () => {
+        cy.get(
+          '#DatasetExplorerChart div[class*="rotatedVerticalBox"] span[class*="textContainer"]'
+        ).contains(defaultYAxis);
+      });
+      if (!noY) {
+        it("should change to different y-axis title", () => {
+          cy.get(
+            '#DatasetExplorerChart div[class*="rotatedVerticalBox"] button'
+          ).click();
+
+          cy.get(
+            "#AxisConfigPanel div[class*='ms-ChoiceFieldGroup'] label:eq(3)"
+          )
+            .invoke("text")
+            .then((text1) => {
+              cy.get(`#AxisConfigPanel label:contains(${text1})`).click();
+              cy.get("#AxisConfigPanel")
+                .find("button")
+                .contains("Select")
+                .click();
+              cy.get(
+                '#DatasetExplorerChart div[class*="rotatedVerticalBox"] button:eq(0)'
+              ).contains(text1);
+            });
+          cy.get(
+            '#DatasetExplorerChart div[class*="rotatedVerticalBox"] button'
```

Comment:
I see some of these strings repeated, like:
```
#DatasetExplorerChart div[class*="rotatedVerticalBox"]
```

do you think it might be better to move the repeating strings to a constants file, to make it easier to maintain/update tests in the future?

Reply:
or even just at top of the file and not a separate constants file if they only repeat within the test

Reply:
Thanks for the comment, moved all repeating strings to constant file.